Source/WebCore/css/parser/CSSParserToken.cpp

```diff
@@ -753,7 +753,7 @@ void CSSParserToken::serialize(StringBuilder& builder, const CSSParserToken* nex
         break;
     case NonNewlineWhitespaceToken: {
         auto count = mode == SerializationMode::CustomProperty ? m_whitespaceCount : 1;
-        for (auto i = 0u; i < count; ++i)
+        for (size_t i = 0; i < count; ++i)
```

Comment:
`m_whitespaceCount` is `unsigned`, so this should also be `unsigned`.
```suggestion
        for (unsigned i = 0; i < count; ++i)
```
or
```suggestion
        for (decltype(count) i = 0; i < count; ++i)
```

Reply:
Resorted to `decltype(count)`